benchmark/benchncnn.cpp

```diff
@@ -240,6 +240,8 @@ int main(int argc, char** argv)
     ncnn::set_omp_dynamic(0);
     ncnn::set_omp_num_threads(num_threads);
 
+    ncnn::set_denormals_zero();
```

Comment:
It is fine to ask user to enable it explicitly.
I think we can do better for the api usage.

Add a switch to the Option class, and make it be true in constructor.
In Extractor::extract function, check the option flag and enable it when needed, and restore it after inference.
You may refer how the opt.openmp_blocktime flag is handled.
So, users would get the speed optimization automatically without changing any code.